add mathjax dependency

catethos opened the pull request:
added mathjax dependency
see https://github.com/HubPress/dev.hubpress.io/issues/15

catethos committed:
add mathjax dependency

anthonny commented on src/helpers/ghost_foot.js:
```diff
@@ -44,7 +44,11 @@ ghost_foot = function (options) {
-
+    foot.push(
+    `
+    <script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'</script>
```
Could you add a `>` between `HTMLorMML'` and `</script>` please?
Like this

```
<script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>
```

anthonny commented:
Thank you so much for this PR and sorry for having take so much to answer.
I made a test, it works great on the generated content.
The next step (not in this PR) is to enable MathJax on the preview

catethos committed:
Update ghost_foot.js

added ">" in 
foot.push(
    `
    <script src='https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'></script>
    `
    );

anthonny commented:
Thanks again! 👍
